modules/flight_interface/flight_interface_worker.py

```diff
@@ -45,4 +50,11 @@ def flight_interface_worker(
         if not result:
             continue
 
+        # Attempt to remove latest odometry data from queue 
```

Comment:
Explain the goal, not just what the specific line does. For example: `# Replace any existing (old) odometry data with the latest`